### PR TITLE
Add ROSCon2019 post.

### PR DESCRIPTION
**Pull request by BorjaOuterelo**

### CODE REVIEW

File: _posts/2019-11-01-micro-ROS-at-ROSCon2019.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: micro-ROS at ROSCon 2019 Macau
+author: borjaouterelo 
+---
+
+![ROSCon2019Macu](/img/posts/ROSConMacau.png){:.img-responsive and style="max-width: 50%; margin-left: auto; margin-right: auto;"}
```
Comment (ralph-lange) [inline _posts/2019-11-01-micro-ROS-at-ROSCon2019.md]:
```suggestion
![ROSCon2019Macau](/img/posts/ROSConMacau.png){:.img-responsive and style="max-width: 50%; margin-left: auto; margin-right: auto;"}
```

File: _posts/2019-11-01-micro-ROS-at-ROSCon2019.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ROSCon ([https://roscon.ros.org](https://roscon.ros.org/)) is the most important event of the ROS community. 
+The 2019 edition was held in Macau, China.
+This edition has been the biggest to date with more than 500 attendants.
```
Comment (ralph-lange) [inline _posts/2019-11-01-micro-ROS-at-ROSCon2019.md]:
```suggestion
This edition has been the biggest to date with about 700 attendants.
```
Karsten Knese mentioned about 700 participants.

File: _posts/2019-11-01-micro-ROS-at-ROSCon2019.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+During this year's event, micro-ROS was present in two different ways a talk and booths on the exhibition floor.
```
Comment (ralph-lange) [inline _posts/2019-11-01-micro-ROS-at-ROSCon2019.md]:
```suggestion
During this year's event, micro-ROS was present in two different ways: a talk and booths on the exhibition floor.
```

File: _posts/2019-11-01-micro-ROS-at-ROSCon2019.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Talk
+
+![Ingo&BorjaPresentation](https://pbs.twimg.com/media/EIRm9RPXYAAKwnN?format=jpg&name=large){:.img-responsive and style="max-width: 60%; float: right; padding-left: 10px"}
+During the second day, Ingo and Borja presented micro-ROS to the ROS community.
+They presented the current status of the framework and supplement with some rationale on why to use it and why we need microcontrollers in Robots.
```
Comment (ralph-lange) [inline _posts/2019-11-01-micro-ROS-at-ROSCon2019.md]:
```suggestion
They presented the current status of the framework and supplement with some rationale on why to use it and why we need microcontrollers in robots.
```

File: _posts/2019-11-01-micro-ROS-at-ROSCon2019.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+## Exhibition
+
+Within the Exhibition floor, there were two booths with material directly related to micro-ROS, Bosch Corporate Research booth, and eProsima's.
```
Comment (ralph-lange) [inline _posts/2019-11-01-micro-ROS-at-ROSCon2019.md]:
```suggestion
Within the exhibition floor, there were two booths with material directly related to micro-ROS, Bosch Corporate Research booth, and eProsima's.
```

File: _posts/2019-11-01-micro-ROS-at-ROSCon2019.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+In the Bosch booth, they had the next evolution of the Community demo, Kobuki, with different hardware setup and some premier of the upcoming updates to come.
```
Comment (ralph-lange) [inline _posts/2019-11-01-micro-ROS-at-ROSCon2019.md]:
```suggestion
In the Bosch booth, they had the next evolution of the micro-ROS community demo based on a Kobuki platform with different hardware setup and some premier of the upcoming updates to come.
```

File: _posts/2019-11-01-micro-ROS-at-ROSCon2019.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+The eProsima booth was divided into two different spaces, one dedicated to Fast RTPS, their well known DDS implementation powering ROS 2 communications.
+The other space was dedicated to microcontrollers development with Micro XRCE-DDS and micro-ROS as main attractions.
+In this "micro" area, there was a TV showing a reel presentation of micro-ROS, the video prepared by FIWARE marketing team, and PIAP's video on benchmarking.
+Also, in this area, and grabbing almost all the attraction of the public, there were two different micro-ROS demonstrations:
+* A first demonstration using a Crazyflie 2.1 and sending data to the RVIz, which was visualized on the TV screen.
+* A second Crazyflie 2.1 issuing commands to a TurtleBot 2 platform to move it.
```
Comment (ralph-lange) [inline _posts/2019-11-01-micro-ROS-at-ROSCon2019.md]:
```suggestion
* A second Crazyflie 2.1 issuing commands to a Kobuki platform to move it.
```

File: _posts/2019-11-01-micro-ROS-at-ROSCon2019.md (hunk 2 above)
Comment (BorjaOuterelo) [reply _posts/2019-11-01-micro-ROS-at-ROSCon2019.md]: Thanks, I was looking for the data but haven't found official numbers. Also interesting will be the streaming viewers.

**Pull request merged**